app/controllers/review_mapping_controller.rb

```diff
@@ -22,25 +20,22 @@ def action_allowed?
           'assign_quiz_dynamically',
           'start_self_review'
       true
-    else current_user_has_instructor_privileges?
```

Comment:
Is there a reason why this code was removed?

Reply:
Reverted this change, as it was incorrectly taken from master to beta.